test/fixtures/linter/rules/NoDeprecatedApi/NoDeprecatedApi_negative.js

```diff
@@ -1,14 +1,11 @@
+const { default: Button } = require("sap/m/Button");
```

Comment:
Was this new line intentional?

The snapshot shows the following error `Unable to analyze this method call because the type of identifier in "require("sap/m/Button")" could not be determined`